build/dockerfiles/Dockerfile

```diff
@@ -14,7 +13,7 @@ COPY . .
 RUN \
     CGO_ENABLED=0 go build \
     -o kit \
-    -ldflags="-s -w -X kitops/pkg/lib/constants.Version=${version} -X kitops/pkg/lib/constants.GitCommit=$gitCommit -X kitops/pkg/lib/constants.BuildTime=$(date -u +'%Y-%m-%dT%H:%M:%SZ')"
+    -ldflags="-s -w -X kitops/pkg/lib/constants.Version=${version} -X kitops/pkg/lib/constants.GitCommit=$(git rev-parse --short HEAD) -X kitops/pkg/lib/constants.BuildTime=$(date -u +'%Y-%m-%dT%H:%M:%SZ')"
```

Comment:
Should we do a build here or download from releases? Downloading would be make it easier for us to correlate.

Reply:
We could download, but then we'd need an annoying arch switch and it'd be harder to do `next`-style image builds.

I suppose it would be useful to have the build timestamp match up though. Maybe we do both?